Added initial setup for dealer header component

helloimela opened the pull request:
- Added variation to the header
- Added c-dealer-header as a separate component

**Solving issue**</br>
Fixes: #116 


**Additional context**<br/>
To review, checkout on Scania theme branch https://github.com/scania/scania-theme/pull/72

Commit:
Added initial setup for dealer header component

- Added variation to the header
- Added c-dealer-header as a separate component

mmexvr commented on src/components/dealer-header/dealer-header.tsx:
```diff
@@ -0,0 +1,68 @@
+})
+export class HelloWorld {
+  /** Dealer name */
+  @Prop() dealerName: string;
```
Might want to use what regular header uses for site-name instead of dealer-name?

Same with site-name and dealer-site

This is a perference question, so we do the same for similare situations

mmexvr reviewed (changes requested):
Need the intital testing for the component, like header and so on both e2e and spec

Commit:
Merge branch 'master' into feature/add_dealer_header

Commit:
Added e2e & unit testing

- Added unit testing
- Added e2e testing
- Fix class name in dealer-header.tsx

Commit:
Update dealer component attributes

- Make it consistent with c-header
- Update e2e test attribute
- Update example

mmexvr commented:
Merged this when scania-theme is released scania/scania-theme#72
